portfolio/src/main/java/com/google/sps/servlets/NewServlet.java

```diff
@@ -8,12 +8,12 @@
 import javax.servlet.http.HttpServletResponse;
 
 
-@WebServlet("/servlet")
+@WebServlet("/step_one_servlet")
 public class NewServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Is it working?</h1>");
+    response.getWriter().println("Is it working?");
```

Comment:
Try to not leave debug messages in code that is being merged (unless this is what you wanted it to say).